### DISCUSSION TITLE
there is difference between css from the gem and when I put it manually

### DISCUSSION THREAD

**Discussion by laptopmutia** [josefarias/hotwire_combobox#139] [General]
so there is option for the css to use `<%= combobox_style_tag %>` or using `*= require hotwire_combobox`

but those two gives difference css compared to when I add hotwire_combobox.css to my `/app/assets/stylesheets/` and copy the code from here https://github.com/josefarias/hotwire_combobox/blob/main/app/assets/stylesheets/hotwire_combobox.css

Idk why
here is with the head tag and adding the css to application.css
<img width="194" alt="Screenshot 2024-04-05 at 03 50 57" src="https://github.com/josefarias/hotwire_combobox/assets/43276109/5a382aaa-692e-4129-8ee4-15dc0c392314">

here is how adding the css manually looks like

<img width="194" alt="Screenshot 2024-04-05 at 03 49 45" src="https://github.com/josefarias/hotwire_combobox/assets/43276109/99b64750-5b98-4310-b5d5-a16ee5c7f729">

**Comment by josefarias** [comment]
@joyoy96 thanks for the report! You're most likely not running the latest version of the gem (v0.2.1). The code you're copying is from the latest version.

Please update your gem to the latest version and let me know if it's not fixed for you. Thanks!
